README.md

```diff
@@ -29,9 +31,9 @@ precision computations (for accuracy requirements that exceeds double precision
 This package can be installed using
 
 ```julia
-using Pkg
-Pkg.add("https://github.com/mikelehu/IRKGaussLegendre.jl")
-using IRKGaussLegendre
+julia>using Pkg
+julia>Pkg.add("IRKGaussLegendre.jl")
+julia>using IRKGaussLegendre
```

Comment:
```suggestion
using Pkg
Pkg.add("IRKGaussLegendre.jl")
using IRKGaussLegendre
```

Reply:
I agree with your suggested change (removing julia>,...). I have created a pull request to understand the GitHub flow.  Once the pull request is accepted, will the changes be merged into the base branch? 
I would also like to tell you that we have proposed a project to a graduate student to add in the implementation of Gauss Legendre event handling and continuous dense output. I hope you find it interesting

Reply:
>we have proposed a project to a graduate student to add in the implementation of Gauss Legendre event handling and continuous dense output. I hope you find it interesting

That sounds great. Have you derived a high order interpolation for GL methods? I don't recall one.

Reply:
I think so.  Do you think we should include any other functionality in IRKGL16?

Reply:
Check compatibility with ForwardDiff for automatic differentiation and I think you've covered everything important. I can definitely help out with AD errors if one find one: that's just pushing generic types through.

Reply:
We don't understand how to add our own interpolation function in IRGL16.
Is there documentation with examples that can help us?

Reply:
You just create a callable struct. Here's a few examples: https://github.com/SciML/SciMLBase.jl/blob/master/src/interpolation.jl

Reply:
We have created a callable struct for own interpolation function (see IRKGLSolver.jl).
[IRKGL16Solver.jl.txt](https://github.com/SciML/IRKGaussLegendre.jl/files/6086110/IRKGL16Solver.jl.txt)

But when we build the solution, should we specify our interpolation function? How?
In this way?
``
  if (dense==true)

     sol= sol=DiffEqBase.build_solution(prob,alg,tt,uu,  destats=destats,retcode= :Success,
        		                    ks=ks ,dense=true,interp=IRKGL16Interp )
else

    sol=DiffEqBase.build_solution(prob,alg,tt,uu, destats=destats,retcode= :Success)

end

``

Reply:
yeah relying on constant folding and doing that is good.

Reply:
When I  try it,  I get the next error: 
`sol = solve(prob,IRKGL16(),dense=true,saveat=1) `
"MethodError: no method matching interp_summary(::Type{IRKGL16Interp})
Closest candidates are:
  interp_summary(!Matched::Nothing) at /home/joseba/.julia/packages/DiffEqBase/elwdl/src/interpolation.jl:38
  interp_summary(!Matched::DiffEqBase.HermiteInterpolation) at /home/joseba/.julia/packages/DiffEqBase/elwdl/src/interpolation.jl:35
  interp_summary(!Matched::DiffEqBase.LinearInterpolation) at /home/joseba/.julia/packages/DiffEqBase/elwdl/src/interpolation.jl:36
  ..."

[Interpolation test.pdf](https://github.com/SciML/IRKGaussLegendre.jl/files/6095939/Interpolation.test.pdf)

Reply:
That's an error from the printing: https://github.com/SciML/SciMLBase.jl/blob/master/src/interpolation.jl#L34

Reply:
Sorry but we are understanding something wrong.
interp_summary(::IRKGL16Interp) function is defined:

`struct IRKGL16Interp{T} <: DiffEqBase.AbstractDiffEqInterpolation
  t::T
  u::T
  ks::Array{T,1} 
end`

`interp_summary(::IRKGL16Interp)="` IRKGL16 `interpolation"`

`(id::IRKGL16Interp)(tvals,idxs,deriv,p,continuity::Symbol=:left)` = interpolation(tvals,id,idxs,deriv,p,continuity)`

Reply:
You have to overload the one in DiffEqBase